var/spack/repos/builtin/packages/libzmq/package.py

```diff
@@ -34,18 +34,25 @@ class Libzmq(AutotoolsPackage):
     variant("drafts", default=False,
             description="Build and install draft classes and methods")
 
+    variant("docs", default=True,
+            description="Build documentation")
+
+    variant("libbsd", default=True,
+            description="Use strlcpy from libbsd " +
+                        "(will use own implementation if false)")
```

Comment:
Is this just a link time modification? Asking since I don't see any use of this variant than in other directives, so was wondering if this was ok.

Reply:
@alalazo if I'm reading the libzmq configuration script correctly, it will detect presence or absence of libbsd automatically, without any flags - if that's what you are asking. Not sure why the dependency type was set to `link` initially, since configure script checks for libbsd presence...

Reply:
If there is a configuration option that **can** be passed I think it's better to pass it. That will prevent any misdetection to cause a discrepancy between the value of the variant and what was really built.

Reply:
@alalazo good idea. Added.